signalr-go-server/pkg/signalr/hubclients.go

```diff
@@ -0,0 +1,51 @@
+package signalr
+
+type HubClients interface {
+	All() ClientProxy
+	Client(connectionID string) ClientProxy
+	Group(groupName string) ClientProxy
+	Caller() ClientProxy
```

Comment:
We don't do caller unfortunately unless we have a per invocation way of passing this state. This is still a gap but we should remove it for now.

Reply:
see code for copying the hub prototype below